app/views/admin/users/_sign_in_table.html.erb

```diff
@@ -10,6 +10,10 @@
       </td>
 
       <td><%= admin_date(sign_in.created_at, ago_only: true) %></td>
+
+      <td>
+        <strong><%= sign_in.other_users.size %></strong> others using this IP
```

Comment:
Probably okay for now, but I think we'll want to handle [Apple Private Relay](https://developer.apple.com/support/prepare-your-network-for-icloud-private-relay/). Probably annotate the stored country with a note so that a high count + APR country wouldn't create suspicion?

Reply:
Might want to add some conditional for when we have no recorded sign ins

Reply:
```suggestion
        <strong><%= pluralize sign_in.other_users.size, 'other' %></strong> using this IP
```

Reply:
> `pluralize sign_in.other_users.size`

I thought about this, but I wanted only the number to be `<strong>`.

Could enhance further with only bolding if number is > 1, but not clear that's useful yet.

Reply:
> Might want to add some conditional for when we have no recorded sign ins

Noted elsewhere.

> Need to return distinct users

Noted elsewhere.